packages/core/src/components/ModalNew/layouts/ModalBasicLayout/__stories__/ModalBasicLayout.stories.tsx

```diff
@@ -31,14 +30,17 @@ export default {
   subcomponents: { ModalBasicLayout, ModalHeader, ModalContent, ModalFooter, ModalFooterWizard, TransitionView },
   argTypes: metaSettings.argTypes,
   decorators: metaSettings.decorators,
-  tags: ["internal"]
+  tags: ["internal"],
+  parameters: {
+    chromatic: {}
+  }
```

Comment:
What's this for?

Reply:
a mistake :)
I want to solve it in another PR, I have an issue where Chromatic captures the Modal story too early